scripts/ansible/roles/linux/openenclave/tasks/ubuntu/packages-install.yml

```diff
@@ -16,13 +16,6 @@
     apt_key_url: "https://apt.llvm.org/llvm-snapshot.gpg.key"
     apt_repository: "{{ llvm_apt_repository }}"
 
-- include_role:
```

Comment:
Why this is not needed?

Reply:
This was a temporary configuration. It's removed in this file: scripts/ansible/roles/linux/common/vars/ubuntu/bionic.yml